pkg/mco/mc.go

```diff
@@ -0,0 +1,244 @@
+package mco
```

Comment:
can this file be renamed as machineconfig.go ? I don't want the brevity of the filename to cause folks to easily overlook it.